shipwright/cache.py

```diff
@@ -3,6 +3,7 @@
 import sys
 import traceback
 
+from docker import errors as d_errors
 from requests import exceptions as requests_exceptions
 
```

Comment:
Yep, that would be cleaner.  Should that be incorporated here, or a separate PR?